pocs/core.py

```diff
@@ -323,7 +323,7 @@ def is_dark(self):
 
         """
         # See if dark
-        is_dark = self.observatory.is_dark
+        is_dark = self.observatory.is_dark()
```

Comment:
It would be nice if we had a naming convention allowing one to distinguish fields from methods.

Reply:
Methods should have begin with a verb, fields should be nouns. Our predicates are a mish-mash right now between properties and methods. I think it makes more sense to make this a property instead of a method but was more consistent the other way. Can change in future (or now I suppose).